discover/index.html

```diff
@@ -124,5 +124,19 @@ <h3 class="npub w3-large">follow the journey on our nostr:</h3>
   }
 }
 </script>
+<!--- nostri.chat widget by pablof7z--->
+<script
+  src="https://nostri.chat/public/bundle.js"
+  <!-- YOUR PUBKEY IN HEX FORMAT -->
+  data-website-owner-pubkey="YOUR_PUBKEY"
+  <!-- THE TYPE OF CHAT YOU WANT: GLOBAL or DMs -->
+  data-chat-type="GLOBAL"
+  <!-- If you use GLOBAL you can choose set a comma-separated list of hashtags-->
+  data-chat-tags="#nostrica,#nostrica2023"
```

Comment:
better do it without the hashtag, that was an error on my sample code I need to correct:

```
  data-chat-tags="nostrica,nostrica2023"
```

Reply:
Thank you, I'll remove the hashtags. I will test whether we can see the feed.

Reply:
Also, does it matter which relays or how many are added to the list? I left your list intact, except for nostrica's.